src/libcrun/linux.c

```diff
@@ -2736,29 +2736,96 @@ libcrun_set_oom (libcrun_container_t *container, libcrun_error_t *err)
   return 0;
 }
 
+const char *sysctlRequiringIPC[] = {
+  "kernel/msgmax",
+  "kernel/msgmnb",
+  "kernel/msgmni",
+  "kernel/sem",
+  "kernel/shmall",
+  "kernel/shmmax",
+  "kernel/shmmni",
+  "kernel/shm_rmid_forced",
+  NULL
+};
+
+static int
+validate_sysctl (const char *original_value, const char *name, unsigned long namespaces_created, libcrun_error_t *err)
+{
+  const char *namespace = "";
+
+  if ((namespaces_created & CLONE_NEWIPC) == 0)
+    {
+      namespace = "IPC";
```

Comment:
I am not entirely sure but should this also filter out `vm/`

Reply:
you are right.  We need to block that as well.  Pushed a new version

Reply:
@kolyshkin this would differ from `runc` but could you confirm as well. If `runc` needs it.

Reply:
@flouthoc I think you are misreading runc code -- it denies everything except for whatever is explicitly allowed. It looks like this code (at least in its current iteration) is doing the same thing.

Reply:
ackd yes.